src/web/app/pages-logs/logs-page.component.ts

```diff
@@ -40,27 +62,37 @@ interface QueryParams {
   selector: 'tm-logs-page',
   templateUrl: './logs-page.component.html',
   styleUrls: ['./logs-page.component.scss'],
+  animations: [collapseAnim],
 })
 export class LogsPageComponent implements OnInit {
   readonly LOGS_RETENTION_PERIOD_IN_DAYS: number = ApiConst.LOGS_RETENTION_PERIOD;
   readonly LOGS_RETENTION_PERIOD_IN_MILLISECONDS: number = this.LOGS_RETENTION_PERIOD_IN_DAYS * 24 * 60 * 60 * 1000;
   readonly SEVERITIES: string[] = ['INFO', 'WARNING', 'ERROR'];
+  readonly EVENTS: string[] = ['REQUEST_RECEIVED', 'RESPONSE_DISPATCHED', 'EMAIL_SENT', 'FEEDBACK_SESSION_AUDIT'];
+  readonly SEVERITY: string = 'severity';
+  readonly MIN_SEVERITY: string = 'minSeverity';
+  readonly EVENT: string = 'event';
+  readonly API_ENDPOINTS: string[] = Object.values(ActionNames).sort();
 
   formModel: SearchLogsFormModel = {
-    logsSeverity: new Set(),
+    logsSeverity: '',
+    logsMinSeverity: '',
+    logsEvent: '',
+    logsFilter: '',
     logsDateFrom: { year: 0, month: 0, day: 0 },
     logsTimeFrom: { hour: 0, minute: 0 },
     logsDateTo: { year: 0, month: 0, day: 0 },
     logsTimeTo: { hour: 0, minute: 0 },
   };
-  previousQueryParams: QueryParams = { searchFrom: '', searchUntil: '', severities: '' };
+  previousQueryParams: QueryParams = { searchFrom: '', searchUntil: '' };
```

Comment:
I know this is not caused by this PR, but is there a reason for naming this field `previous`? Is there a `currentQueryParams` or `nextQueryParams` somewhere, or planned to be?

Reply:
When users load more logs, the request parameters need to be the same as the previous request. I named it this way because this field stores the previous request and use it for the current request. But since there is no `currentQueryParams` or `nextQueryParams`, I will change it to just `queryParams`